eclass/cmake-utils.eclass

```diff
@@ -116,16 +116,18 @@ inherit toolchain-funcs ninja-utils flag-o-matic multiprocessing xdg-utils
 case ${EAPI} in
 	[56])
 		: ${CMAKE_MAKEFILE_GENERATOR:=emake}
+		_cmake_eqawarn_command=eqawarn
 		inherit eapi7-ver eutils multilib
 		;;
 	*)
 		: ${CMAKE_MAKEFILE_GENERATOR:=ninja}
+		_cmake_eqawarn_command=die
```

Comment:
To be honest, this wasn't exactly my intent. I meant making existing QA warnings fatal in EAPI 7 but starting with eqawarn for future warnings ;-).

Reply:
Please do it yourself then, this has cost me too much time already